server/models/task.coffee

```diff
@@ -0,0 +1,22 @@
+CONFIG = require 'config'
+__ = CONFIG.universalPath
+_ = __.require 'builders', 'utils'
+error_ = __.require 'lib', 'error/error'
+
+module.exports = Task = {}
+
+tests = require './tests/task'
+
+Task.create = (suspect, suggestion)->
+  _.types arguments, [ 'string', 'string' ]
+
+  tests.pass 'suspect', suspect
+
+  now = Date.now()
+
+  return task =
+    type: "task"
```

Comment:
it might be a bit anticipating, but, additionally to this type 'task' (which might actually not be needed if we agree to put only documents of type tasks in the tasks database, which is now the [recommended way to do things](https://github.com/inventaire/inventaire/issues/85)), we could have a subtype to explicitly describe the kind of task we are dealing with - here something with "deduplication". The alternative would be to duck-type the kind of task from the task-specific attributes (here, possilbly `suspectUri` and `suggestionUri`), but expliciting it feels cleaner right?